docs/mlflow_integration.md

```diff
@@ -0,0 +1,67 @@
+## MLFlow Integration
+YOLOX now supports MLFlow integration. MLFlow is an open-source platform for managing the end-to-end machine learning lifecycle. It is designed to work with any ML library, algorithm, deployment tool, or language. MLFlow can be used to track experiments, metrics, and parameters, and to log and visualize model artifacts. \
+For more information, please refer to: [MLFlow Documentation](https://www.mlflow.org/docs/latest/index.html)
+
+## Follow these steps to start logging your experiments to MLFlow:
+### Step-1: Install MLFlow via pip 
+```bash
+pip install mlflow python-dotenv
+```
+
+### Step-2: Set up MLFlow Tracking Server
+Start or connect to a MLFlow tracking server like databricks. You can start a local tracking server by running the following command:
+```bash
+mlflow server --host 127.0.0.1 --port 8080
+```
+Read more about setting up MLFlow tracking server [here](https://mlflow.org/docs/latest/tracking/server.html#mlflow-tracking-server)
+
+### Step-3: Set up MLFlow Environment Variables
+Set the following environment variables in your `.env` file:
+```bash
+MLFLOW_TRACKING_URI="127.0.0.1:5000"  # set to your mlflow server URI
+MLFLOW_EXPERIMENT_NAME="/path/to/experiment"  # set to your experiment name
+MLFLOW_TAGS={"release.candidate": "DEV1", "release.version": "0.0.0"}
+# config related to logging model to mlflow as pyfunc
+YOLOX_MLFLOW_LOG_MODEL_ARTIFACTS="True" # whether to log model or not 
+YOLOX_MLFLOW_LOG_HISTORICAL_MODEL_PER_n_EPOCHS=30 # log model after every n epochs
```

Comment:
Should be YOLOX_MLFLOW_LOG_MODEL_PER_n_EPOCHS

Reply:
@Im-Himanshu  Please update your doc

Reply:
Done